examples/computer_vision/unets_tf_keras/model_def.py

```diff
@@ -140,7 +140,7 @@ def build_validation_data_loader(self):
             split="test",
             with_info=True,
             data_dir=self.context.get_data_config()['data_dir'],
-            download=False
+            download=True,
```

Comment:
Does this mean "allow it to be downloaded if it is missing" or "always download"?

Reply:
This is "always download" right now, but I can change that to "allow to be download if missing".